src/sentry/issues/attributes.py

```diff
@@ -192,6 +203,8 @@ def _bulk_retrieve_snapshot_values(
             "group_id": group_value.id,
             "status": group_value.status,
             "substatus": group_value.substatus,
+            "priority": group_value.priority,
+            "first_release": group_value.first_release,
```

Comment:
This should be `first_release_id` so that when we have a `Group` it works correctly,  (i think a group is passed here sometimes?). So I guess we need to rename `GroupValues.first_release`?

if it's a huge pain then feel free to just skip this review though

Reply:
ahh that makes sense, updated!